script/Ciel/Day 8/08_01_CIEL08_1C.txt

```diff
@@ -209,7 +209,7 @@ Arihiko and I shoot each other a glance, bewildered by her unreasonable attitude
 -- Page 82, Offset 18460. Glued. Mods: @x.
 -- よし、そういうコトなら力になりましょう！　遠野、しりとりしようぜ！」
  Alrighty, if that's how it is, lemme lend you a hand!
-  Tohno, let's play shiritori!"
+ Tohno, let's play shiritori!"
```

Comment:
revert, this line is glued

Reply:
Nah, this one's correct - there were 2 spaces instead of 1